packages/core/src/components/Banner.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from "react";
+import {
+  Button,
+  Text,
+  View,
+  ViewStyle,
+  StyleSheet,
+  StyleProp,
+  Animated,
+} from "react-native";
+import Surface from "./Surface";
+import type { IconSlot } from "../interfaces/Icon";
+import shadow from "../styles/shadow";
+import { withTheme } from "../theming";
+import type { Theme } from "../styles/DefaultTheme";
+
+const ELEVATION = 1;
+const DEFAULT_MAX_WIDTH = 960;
+
+type Props = {
+  initiallyVisible: boolean;
+  dismissable: boolean;
+  buttonColor?: string;
+  icon?: string;
+  content: string;
+  contentStyle?: StyleProp<ViewStyle>;
+  style?: StyleProp<ViewStyle>;
+  ref?: React.RefObject<View>;
+  /**
+   * @optional
+   */
+  theme: Theme;
+} & IconSlot;
+
+type NativeEvent = {
+  nativeEvent: {
+    layout: {
+      x: number;
+      y: number;
+      width: number;
+      height: number;
+    };
+  };
+};
+
+const Banner: React.FC<Props> = ({
+  initiallyVisible,
```

Comment:
Not passing in a value might work, but can you be explicit about non-optional props and set to false?

Reply:
Certainly, that makes sense